src/Inkluzitron/Program.cs

```diff
@@ -67,32 +72,23 @@ static private async Task MainAsync(string[] args)
                 .AddSingleton<ReactionSettings>()
                 .AddSingleton<BdsmTestOrgSettings>()
                 .AddSingleton<ReactionsModule>()
-                .AddSingleton<ProfilePictureService>()
                 .AddSingleton<FontService>()
                 .AddSingleton<GraphPaintingService>()
                 .AddSingleton<UserBdsmTraitsService>()
                 .AddSingleton<BdsmTraitOperationCheckTranslations>()
                 .AddSingleton<ImagesService>()
+                .AddSingleton(new FileCache(cacheDirLocation))
                 .AddHttpClient()
-                .AddMemoryCache();
-
-            services.AddLogging(config =>
-            {
-                config
-                    .SetMinimumLevel(LogLevel.Information)
-                    .AddSystemdConsole(opt =>
+                .AddMemoryCache()
+                .AddLogging(config =>
+                {
+                    config.AddConfiguration(configuration.GetSection("Logging"));
```

Comment:
Nenačítá se to automaticky? :hmmNote:

Reply:
Myslel jsem si že jo, ale bez toho to printí všechno :(